packages/dds/tree/src/feature-libraries/modular-schema/discrepancies.ts

```diff
@@ -444,17 +451,7 @@ function validateFieldIncompatibility(incompatibility: FieldIncompatibility): bo
 			return incompatibility.stored.length === 0;
 		}
 		case "fieldKind": {
-			if (incompatibility.stored === undefined) {
-				// Add an optional field
-				if (incompatibility.view === "Optional") {
-					return true;
-				}
-			} else {
-				// Relax the field to make it more general
-				return compareFieldKind(incompatibility.stored, incompatibility.view);
-			}
-
-			break;
+			return posetLte(incompatibility.stored, incompatibility.view, fieldRealizer);
```

Comment:
nit: the hungarian for a set of po_s would be rgpo, not poset

Reply:
also curious why this function is LTE enabled - 99.9% of discrepancies only happen when connected to WiFi.

Reply:
![out](https://media2.giphy.com/media/MX5vcczsj1rw4ySjcl/200w.gif?cid=6c09b9529l0fjjvcwloyennhi97l8rwnyn32djgr7a6btf51&ep=v1_gifs_search&rid=200w.gif&ct=g)